python/src/deltachat/chat.py

```diff
@@ -109,6 +109,30 @@ def get_join_qr(self):
 
     # ------  chat messaging API ------------------------------
 
+    def send_msg(self, msg):
```

Comment:
I added this because I needed to be able to send a message after `Message.new_empty().set_file()` without going via the dc_prepare_msg logic since using dc_prepare_msg wants the file to be in the blobdir instead of copying it there.

Reply:
This could potentially replace `send_prepared` since I think it does the same thing now.  Though I didn't do that in this PR yet.